### DISCUSSION TITLE
Quasar q-table

### DISCUSSION THREAD

**Discussion by priyanka-kurve** [quasarframework/quasar#13256] [General - Components / Directives / etc] [answered]
Can anyone help in How to make dynamic editable rows in quasar q-table ?

![n](https://user-images.githubusercontent.com/60501475/165084369-542738a9-8a0c-4ecc-a99b-0aa793f1e694.jpg)

**Comment by pratik227** [comment, marked as answer]
https://quasar.dev/vue-components/table#popup-editing
